### PR TITLE
Feature/pdct 937 change the schema to remove unused link tables

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Bump dbclient to 3.4.0
- Drivebys for more types of changes in PR template, makefile setup, and readme

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,6 +14,9 @@ Please select the option(s) below that are most relevant:
 - [ ] Bug fix
 - [ ] New feature
 - [ ] Breaking change
+- [ ] GitHub workflow update
+- [ ] Documentation update
+- [ ] Remove legacy code
```
Comment (olaughter) [inline .github/pull_request_template.md:19]: "Refactor" might be a better term for this one to cover adding in replacement code?

**Review by olaughter — APPROVED** [review]
lgtm, nice to see code getting tidier!

**Review by diversemix — APPROVED** [review]
lgtm

File: .github/pull_request_template.md (hunk 1 above)
Comment (katybaulch) [reply .github/pull_request_template.md:19]: Happy with this change!

**Pull request merged**